astro/src/content/docs/_shared/_licensed-edition-blurb.mdx

```diff
@@ -0,0 +1,6 @@
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import EditionBlurb from './_edition-blurb.astro';
+
+<EditionBlurb>
+  This feature is available to licensed FusionAuth instances as of version 1.52.0. A free license is available in the <Breadcrumb>Plan</Breadcrumb> tab for any user who registers in the <a href="https://account.fusionauth.io/" target="_blank">account portal</a>.
```

Comment:
This calls out `1.52.0`, but that version number will change in future releases. Should we make it a `prop`?